js/index.js

```diff
@@ -0,0 +1,38 @@
+function render(pattern, getColor, scale = 100) {
+	const logo = document.getElementById('rs-logo');
+	logo.style.display = 'none';
```

Comment:
It's not very comfortable to manipulate with DOM using style property. I suggest you to create a css class with property display:none, and just add this class to elements if you want to hide it.